tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -19,81 +21,158 @@
-		const branchIsForkedText =
-			'<div id="jetpack-live-branches">' +
-			'<h2>Jetpack Live Branches</h2>' +
-			"<p><strong>This branch can't be tested live because it comes from a forked version of this repo.</p>" +
-			'</div>';
-		const branchIsMergedText =
-			'<div id="jetpack-live-branches">' +
-			'<h2>Jetpack Live Branches</h2>' +
-			'<p><strong>This branch is already merged.</p>' +
-			'</div>';
+
 		if ( ! branchIsForked && ! branchIsMerged ) {
-			appendHtml( markdownBody, canLiveTestText );
+			const link = getLink();
+			const contents = `
+				<h4>Settings</h4>
+				${ getOptionsList(
+					[
+						{
+							label: 'A shortlived site',
+							name: 'shortlived',
+						},
+						{
+							checked: true,
+							label: '<code>WP_DEBUG</code> and <code>WP_DEBUG_LOG</code> set to true',
+							name: 'wp-debug-log',
+						},
+						{
+							label: 'Multisite based on subdomains',
+							name: 'subdomain_multisite',
+						},
+						{
+							label: 'Multisite based on subdirectories',
+							name: 'subdir_multisite',
+						},
+						{
+							label: 'Pre-generate content',
+							name: 'content',
+						},
+						{
+							label: '<code>xmlrpc.php</code> unavailable',
+							name: 'blockxmlrpc',
+						},
+					],
+					100
+				) }
+				<h4>Plugins</h4>
+				${ getOptionsList(
+					[
+						{
+							label: 'WordPress Beta Tester',
+							name: 'wordpress-beta-tester',
+						},
+						{
+							label: 'Gutenberg',
+							name: 'gutenberg',
+						},
+						{
+							label: 'Classic Editor',
+							name: 'classic-editor',
+						},
+						{
+							label: 'WooCommerce',
+							name: 'woocommerce',
+						},
+						{
+							label: 'WooCommerce Beta Tester',
+							name: 'woocommerce-beta-tester',
+						},
+						{
+							label: 'Config Constants',
+							name: 'config-constants',
+						},
+						{
+							label: 'Code Snippets',
+							name: 'code-snippets',
+						},
+						{
+							label: 'WP Rollback',
+							name: 'wp-rollback',
+						},
+						{
+							label: 'WP Downgrade',
+							name: 'wp-downgrade',
+						},
+						{
+							label: 'WP Super Cache',
+							name: 'wp-super-cache',
+						},
+						{
+							label: 'WP Log Viewer',
+							name: 'wp-log-viewer',
+						},
+						{
+							label: 'WP Job Manager',
+							name: 'wp-job-manager',
+						},
+					],
+					33
+				) }
+				<p>
+					<a id="jetpack-beta-branch-link" target="_blank" rel="nofollow noopener" href="${ link }">
+						${ link }
+					</a>
+				</p>
+			`;
+			appendHtml( markdownBody, contents );
 		} else if ( ! branchIsMerged ) {
-			appendHtml( markdownBody, branchIsForkedText );
+			appendHtml(
+				markdownBody,
+				"<p><strong>This branch can't be tested live because it comes from a forked version of this repo.</strong></p>"
+			);
 		} else {
-			appendHtml( markdownBody, branchIsMergedText );
+			appendHtml( markdownBody, '<p><strong>This branch is already merged.</strong></p>' );
+		}
+
+		function getLink() {
```

Comment:
Instead of modifying the URL, we just re-generate it on each time. Feels more straightforward code-wise.